llvm/include/llvm/Transforms/Yk/ModuleClone.h

```diff
@@ -0,0 +1,13 @@
+#ifndef LLVM_TRANSFORMS_YK_MODULE_CLONE_H
+#define LLVM_TRANSFORMS_YK_MODULE_CLONE_H
+
+#include "llvm/Pass.h"
+
+#define YK_CLONE_PREFIX "__yk_clone_"
```

Comment:
Can we document somewhere (maybe in a doc comment at the top of the pass cpp file?) what the cloned versions are? Are they intended to be the optimised versions, or the unoptimisaed versions?

Reply:
sounds good, will add.

Reply:
Added doc comment :point_right: 71f197364bc4b854555c074428fae32cf7fc3de2